lib/matplotlib/rcsetup.py

```diff
@@ -413,6 +413,37 @@ def validate_font_properties(s):
     return s
 
 
+def validate_mathtext_fallback_to_cm(b):
```

Comment:
new validators should be private, i.e. underscore-prefixed (we're slowly making the old ones private too) -- see parenthetical note in https://github.com/matplotlib/matplotlib/pull/16483 as to why.